src/app/components/home/home.module.ts

```diff
@@ -0,0 +1,39 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FeaturedChallengesComponent } from './featured-challenges/featured-challenges.component';
+import { HomemainComponent } from './homemain/homemain.component';
+import { PartnersComponent } from './partners/partners.component';
+import { RulesComponent } from './rules/rules.component';
+import { TestimonialsComponent } from './testimonials/testimonials.component';
+import { TwitterFeedComponent } from './twitter-feed/twitter-feed.component';
+import { HomeComponent } from './home.component';
+import { NgxTwitterTimelineModule } from 'ngx-twitter-timeline';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  declarations: [
+    FeaturedChallengesComponent,
+    HomemainComponent,
+    PartnersComponent,
+    RulesComponent,
+    TestimonialsComponent,
+    TwitterFeedComponent,
+    HomeComponent
+  ],
+  imports: [
+    CommonModule,
+    RouterModule,
+    NgxTwitterTimelineModule
+  ],
+  exports: [
+    FeaturedChallengesComponent,
+    HomemainComponent,
+    PartnersComponent,
+    RulesComponent,
+    TestimonialsComponent,
+    TwitterFeedComponent,
+    HomeComponent
+  ],
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
```

Comment:
What does it do ?

Reply:
Basically, it defines a schema that will allow any non-Angular elements with 'a -' in their name. Without this, all the selector tags were throwing error of being undefined.